app/components/post_body/index.js

```diff
@@ -61,9 +61,10 @@ function makeMapStateToProps() {
         const roles = getCurrentUserId(state) ? getCurrentUserRoles(state) : '';
         const isAdmin = checkIsAdmin(roles);
         const isSystemAdmin = checkIsSystemAdmin(roles);
+        const channelIsArchived = ownProps.hasOwnProperty('channelIsArchived') ? ownProps.channelIsArchived : channel?.delete_at !== 0; //eslint-disable-line camelcase
```

Comment:
If you checking the channel, there is no reason why you need to determine if ownprops is defined, we should look at the root of the problem as of why it is undefined at times.

Reply:
True, it is not being passed to PostBody